src/calculateRentalCost.js

```diff
@@ -29,6 +29,17 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const total = days * 40;
+
+  if (days >= 7) {
+    return total - 50;
+  }
+
+  if (days >= 3) {
+    return total - 20;
+  }
+
+  return total;
```

Comment:
Good job!
Fix just this one. Make some variables for values, because for now we don't understand what it is - 40, 50, 20

Reply:
Thanks for review, Server!